registry/remote/example_test.go

```diff
@@ -496,12 +497,8 @@ func ExampleRepository_Tag() {
 
 	// tag a manifest referenced by the exampleDigest below
 	exampleDigest := "sha256:00e5ffa7d914b4e6aa3f1a324f37df0625ccc400be333deea5ecaa199f9eff5b"
-	descriptor, err := repo.Resolve(ctx, exampleDigest)
-	if err != nil {
-		panic(err)
-	}
 	tag := "latest"
-	err = repo.Tag(ctx, descriptor, tag)
```

Comment:
The scenarios are different. Could we have two examples? One for `Tag` and another for `TagReference`?

Reply:
Kept the original test for Tag, and created a new test for TagReference.